src/components/application_manager/rpc_plugins/rc_rpc_plugin/src/rc_app_extension.cc

```diff
@@ -123,7 +125,43 @@ void RCAppExtension::SaveResumptionData(
 
 void RCAppExtension::ProcessResumption(
     const smart_objects::SmartObject& saved_app,
-    resumption::Subscriber subscriber) {}
+    resumption::Subscriber subscriber) {
+  LOG4CXX_AUTO_TRACE(logger_);
+
+  if (!saved_app.keyExists(
+          application_manager::strings::application_subscriptions)) {
+    LOG4CXX_DEBUG(logger_, "application_subscriptions section is not exists");
+    return;
+  }
+
+  const smart_objects::SmartObject& resumption_data =
+      saved_app[application_manager::strings::application_subscriptions];
+
+  if (!resumption_data.keyExists(message_params::kModuleData)) {
+    LOG4CXX_DEBUG(logger_, "kModuleData section does not exist");
+    return;
+  }
+
+  auto& module_data = resumption_data[message_params::kModuleData];
+
+  const auto lenght = module_data.length();
+
+  if (!lenght) {
```

Comment:
if (!module_data.length())

Reply:
@LuxoftAKutsan 
Fixed in:https://github.com/LuxoftSDL/sdl_core/pull/99/commits/114a2b4f4858dd189f26c13f00aa4292489b374b